src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -240,8 +242,9 @@
       <TestCommandLines Include="copy /y $(_TestILCFolder)\default.rd.xml  %EXECUTION_DIR%" />
       <TestCommandLines Include="rmdir /S /Q %EXECUTION_DIR%int" />
       <TestCommandLines Include="rmdir /S /Q %EXECUTION_DIR%native" />
+      <TestCommandLines Condition="'$(EnableMultifileTesting)' == 'true'" Include="call %RUNTIME_PATH%\TestILC\ilc.exe -buildSharedAssemblies -buildtype $(ILCBuildType) -v diag"/>
```

Comment:
Are you sure you want to do this of every test ? Ideally  this need to be done after the package is pulled from ILC . We need this happen only once , buildSharedAssemblies can take 2 - 4  minutes depending on the machine.